src/view/MainWindow.vala

```diff
@@ -23,12 +23,19 @@ class MainWindow : Gtk.ApplicationWindow {
     private TaskManager task_manager;
     private TaskTimer task_timer;
     private SettingsManager settings;
+    private bool use_header_bar;
```

Comment:
I really like the fact that you have defined a variable for this purpse, so that it can be altered even if Gtk 3.10 was available. This is important for other desktop environments like Unity or KDE.
Great Job :+1: